app/models/user.rb

```diff
@@ -232,10 +234,6 @@ def self.requested_sync_approval
     where(sync_approval_status: :requested)
   end
 
-  def has_role?(*roles)
```

Comment:
Curious, what was this 🤔 ?

Reply:
A quick `blame` points to an initial permissions system. Looks like when we used to use `User#role` more formally, this was a convenience method for authorization checks. Search for `has_role?` in [this PR](https://github.com/simpledotorg/simple-server/pull/643/files)

We don't use `role` formally anymore, and this method isn't invoked anywhere anymore, so I've trashed i.t